node_modules/col-assets/tests/util.js

```diff
@@ -100,6 +110,16 @@ var assertAsset = module.exports.assertAsset = function(asset, expectedAsset, ex
       assert.strictEqual(asset.views, expectedAsset.views);
     }
 
+    // Ensure that the expected categories are present
+    if (expectedCategories) {
```

Comment:
It wouldn't be in line with the rest of the function. The first if checks the provided asset itself, whilst the second if verifies it against the expected asset.